generators/app/templates/src/app/components/ProviderWrapper/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useReducer } from 'react';
+
+interface Props<U extends {}> {
+  Context: React.Context<any>;
+  reducer: React.Reducer<U, { type: string }>;
+  initialState: U;
+}
+
+const withProvider = <T extends any, U>({ Context, reducer, initialState }: Props<U>) => (
```

Comment:
What is this `extends any` doing?

Reply:
I'll change it to {}, I was using it to pass as prop type for the wrapped component